manifests/plugin/ansible.pp

```diff
@@ -40,7 +38,6 @@
   Stdlib::Absolutepath $ansible_dir = $foreman_proxy::plugin::ansible::params::ansible_dir,
   Optional[Stdlib::Absolutepath] $working_dir = $foreman_proxy::plugin::ansible::params::working_dir,
   Boolean $host_key_checking = $foreman_proxy::plugin::ansible::params::host_key_checking,
-  String $stdout_callback = $foreman_proxy::plugin::ansible::params::stdout_callback,
```

Comment:
I think this makes it "Backwards incompatible", strictly speaking, but I am OK with that.

Reply:
just for the record, this is ignored/overriden if you run things via `ansible-runner`, so it wasn't working before already, so it's not a breaking change "as such".